### ISSUE TITLE
ArTaxOr Dataset

### ISSUE THREAD

**Issue by Stlve** [XXX-YES/FSOD#6]
Hello, thank you very much for making your code open source. I would like to ask is that I tried to reproduce the experimental results of the ArTaxOr dataset, but I found that my metric is only 1.41, which is much smaller than the 5.1 in the paper, do you know what the problem is? I have tried both in pre-training weights as r50.pkl and R50-FPN with 3x and the metrics are not very good.

**Comment by woody-panda** [comment]
You can try adjusting the learning rate, see here #5 .
